application/contract/member/member.go

```diff
@@ -160,16 +162,16 @@ func (m *Member) registerNodeCall(rd insolar.Reference, params map[string]interf
 		return nil, fmt.Errorf("incorect input: failed to get 'publicKey' param")
 	}
 
-	return m.registerNode(rd, publicKey, role)
+	return m.registerNode(m.RootDomain, publicKey, role)
```

Comment:
If `registerNode` is method of `m`, why use `m.RootDomain` as first param? Just get `m.RootDomain` inside `registerNode` func